eventtracking/config.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module contains various configuration settings via
+waffle switches for the event-tracking app.
+"""
+
+from edx_toggles.toggles import SettingToggle
+
+# .. toggle_name: SEND_TRACKING_EVENT_EMITTED_SIGNAL
+# .. toggle_implementation: SettingToggle
+# .. toggle_default: False
+# .. toggle_description: When True, the system will publish `TRACKING_EVENT_EMITTED` signals to the event bus. The
+#   `TRACKING_EVENT_EMITTED` signal is emit when a tracking log is emitted.
+# .. toggle_use_cases: circuit_breaker
```

Comment:
[question, non-blocking] I would tend to think of a circuit breaker toggle as being enabled by default. Is the intention to change the default to True once this has been tested in production?

Reply:
Tracking logs can fill up the event bus (on Redis mainly) as those are verbose and are emitted pretty quickly. We are disabling this by default and leaving a setting (empty by default) to allow only a subset of tracking logs to the event bus.

Also, we haven't done any production-like experiment to measure how this will impact Redis and if it will ever break based mainly on the Aspects workflow.

I'm not sure if we ever want to set the default to True as it's only needed for Aspects, or some other custom use case for tracking logs.

@bmtcril what do you think?

Reply:
Might make sense to describe it as `opt_in` instead, then.